libs/common/src/lib/localization/localization.service.ts

```diff
@@ -1,21 +1,31 @@
 import { Injectable, NotFoundException } from "@nestjs/common";
-import { I18nItem, I18nTranslation, LocalizationKey } from "@terramatch-microservices/database/entities";
+import {
+  Form,
+  FormOptionListOption,
+  FormQuestion,
+  FormQuestionOption,
+  FormSection,
+  FormTableHeader,
+  FundingProgramme,
+  I18nItem,
+  I18nTranslation,
+  LocalizationKey
+} from "@terramatch-microservices/database/entities";
 import { Op } from "sequelize";
 import { ConfigService } from "@nestjs/config";
-import { ITranslateParams, normalizeLocale, tx, t } from "@transifex/native";
+import { ITranslateParams, normalizeLocale, tx, t, createNativeInstance } from "@transifex/native";
 import { Dictionary } from "lodash";
 import { ValidLocale } from "@terramatch-microservices/database/constants/locale";
+import { DRAFT, MODIFIED } from "@terramatch-microservices/database/constants/status";
+import { TMLogger } from "../util/tm-logger";
 
 // A mapping of I18nItem ID to a translated value, or null if no translation is available.
 export type Translations = Record<number, string | null>;
 
 @Injectable()
 export class LocalizationService {
-  constructor(configService: ConfigService) {
-    tx.init({
-      token: configService.get("TRANSIFEX_TOKEN")
-    });
```

Comment:
Does the `createNativeInstances` stuff replace the need for this? I'm surprised to see this removed, as I thought it was required by other methods in this service.